### ISSUE TITLE
History email enhancement 

### ISSUE THREAD

**Issue by darthtang** [salesagility/SuiteCRM#759]
My scenario here is that because i archive lots of emails to an account and contacts of the crm i need to click into each record in the history subpanel to see what the email was or if anything is different...which is fine!

But i think it would be cool if you hover over the record in the sub panel and it will display the email body next to your cursor so that you can quickly view the main email of a thread. Suggestions and input as i think this would be useful to lots of SuiteCRM users

**Comment by Mausino** [comment]
:+1:

**Comment by ghermans** [comment]
:+1: +1

**Comment by gunnicom** [comment]
Maybe useful for other objects, too. Like for Notes or Calls etc.

**Comment by darthtang** [comment]
cheers :D

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [History email enhancement](https://trello.com/c/RAgu0cLT/129-history-email-enhancement)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
